src/import-data/ImportData.ts

```diff
@@ -107,10 +107,20 @@ export const ImportData = (props: ImportDataProps): ReactNode => {
     if (workspace.cloudPlatform === 'Azure') {
       // find wds for this workspace
       const wdsUrl = await Ajax().Apps.listAppsV2(workspace.workspaceId).then(resolveWdsUrl);
-      const wdsDataTableProvider = new WdsDataTableProvider(workspace.workspaceId, wdsUrl);
+      const { namespace, name } = workspace;
 
       // call import snapshot
-      await wdsDataTableProvider.importTdr(workspace.workspaceId, importRequest.manifestUrl);
+      const { jobId } = await Ajax().WorkspaceData.startImportJob(wdsUrl, workspace.workspaceId, {
+        url: importRequest.manifestUrl.toString(),
+        type: 'TDRMANIFEST',
+      });
+      const newJob: AzureAsyncImportJob = {
+        targetWorkspace: { namespace, name },
+        jobId,
+        wdsProxyUrl: wdsUrl,
+      };
+      asyncImportJobStore.update((previousJobs) => [...previousJobs, newJob]);
+      notifyDataImportProgress(jobId);
```

Comment:
I don't know how notifying works but it seems odd to explicitly call `notify` when a State object is also changed. Could the notify be triggered by the state change instead?

Reply:
Sorry for being dense, but what State object is being changed here?

Reply:
> what State object is being changed here?

`asyncImportJobStore`. 

Something could `subscribe` to it and show notifications when a new job is added.
https://github.com/DataBiosphere/terra-ui/blob/be17f042533ce0793f62a38ff6b57de90d099887/packages/core-utils/src/state-utils.ts#L38

Reply:
Oh, never mind, I thought that `asyncImportJobStore` was State (I assumed based on its name) but I see it's just an Atom. Sorry!

Reply:
Hm ok I'll leave off setting up the subscribe from this PR since that feels out of scope but not a bad idea.